### DISCUSSION TITLE
Tailwind Integration!

### DISCUSSION THREAD

**Discussion by lnxpy** [collove/pasteme#60] [Announcements]
We are planning to switch from Bootstrap to Tailwind!

**Comment by farhaduneci** [comment]
Great move :handshake:

**Comment by lnxpy** [comment]
## Primary Components for Refactoring

#### 1. There is a static element in the page that contains the asciinema cast. Here is the element:
```html
<script src="https://asciinema.org/a/510373.js" id="asciicast-510373" data-theme="tango" data-size="big" data-speed="3" data-rows="20"></script>
```

#### 2. Do static templates (e.g. images, octocats, logos, and etc) need any change in terms of shape, size, type, and etc? Let me know if so.

#### 3. I'm using [Highlightjs](https://highlightjs.org/) for highlighting. You can find the themes [here](https://github.com/collove/pasteme/tree/development/snippet/static/css/themes). There is also another additional Js component that is responsible for the line number. It's available [here](https://github.com/collove/pasteme/blob/f1253e2c920819212da01b35cf107bdcfccf80f7/snippet/templates/snippet.html#L28).

#### 4. And last but not least, all integrations and refactors are happening to `feature/frontend_refactor`. Make sure to check out.

Good luck. :heart:

**Reply by farhaduneci** [reply]
Thanks for the information,
Nothing static needs to change, all JS tags will remain the same.

Will leaving JS tags as they are handle the highlighting functionality? I do not need to touch any of them.

**Comment by farhaduneci** [comment]
Would you please create an issue for this migration and assign it to me, Sadra :smiley:

**Reply by lnxpy** [reply]
Sure.

**Comment by farhaduneci** [comment]
Hey there, Just wanted to let everyone know **Dark mode** is coming to PasteMe!
Drop your ideas/recommendations please :fire: 

![Demo](https://user-images.githubusercontent.com/53964770/182446720-7ecaf827-7cb1-4027-9678-cb1e9d4d878d.png)

**Reply by lnxpy** [reply]
That's phenomenon! 😍

**Reply by lnxpy** [reply]
Good job Farhad.

**Comment by farhaduneci** [comment]
So many changes applied since last comment, those may be followed in issues section, but I'll post updated here until we move our front-end branch here.

Today I added Octocat images to 404/500 pages,

![image](https://user-images.githubusercontent.com/53964770/184310180-4d294943-9b52-460f-89f7-5d0e02aff6a0.png)
![image](https://user-images.githubusercontent.com/53964770/184310189-4979342c-81f0-4b5f-b760-b15ea9fe9f41.png)

**Reply by lnxpy** [reply]
So fire!! Can't wait for the new release!
